tools/checkpoint_conversion/bert_large_cased_en.ipynb

```diff
@@ -0,0 +1,1221 @@
+        "BERT has been uploaded to TensorFlow Hub. See run_classifier_with_tfhub.py for an example of how to use the TF Hub module, or run an example in the browser on Colab.\"\n",
+        "\n",
+        "### TF Hub statement:\n",
+        "\"The weights of this model are those released by the original BERT authors.\""
+      ],
+      "metadata": {
+        "id": "ByCEoIyn-_Ld"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "text_input = tf.keras.layers.Input(shape=(), dtype=tf.string)\n",
+        "\n",
+        "preprocessor = hub.load(\"https://tfhub.dev/tensorflow/bert_en_cased_preprocess/3\")\n",
+        "tokenizer = hub.KerasLayer(preprocessor.tokenize, name='tokenizer')\n",
+        "tokenized_text = tokenizer(text_input)\n",
+        "\n",
+        "packer = hub.KerasLayer(\n",
+        "    preprocessor.bert_pack_inputs,\n",
+        "    arguments=dict(seq_length=512),\n",
+        "    name='packer')\n",
+        "encoder_inputs = packer([tokenized_text])\n",
+        "\n",
+        "encoder = hub.KerasLayer(\n",
+        "    f\"https://tfhub.dev/tensorflow/bert_en_cased_{MODEL_SPEC_STR}/4\",\n",
+        "    trainable=True)\n",
+        "outputs = encoder(encoder_inputs)\n",
+        "pooled_output = outputs[\"pooled_output\"]      # [batch_size, 1024].\n",
+        "sequence_output = outputs[\"sequence_output\"]  # [batch_size, seq_length, 1024].\n",
+        "\n",
+        "embedding_model = tf.keras.Model(text_input, (pooled_output, sequence_output))"
+      ],
+      "metadata": {
+        "id": "hQ0lMSluxMx1"
+      },
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def preprocess(x):\n",
+        "    tokenizer = keras_nlp.tokenizers.WordPieceTokenizer(\n",
+        "        vocabulary=vocab_path,\n",
+        "        lowercase=False\n",
+        "    )\n",
+        "    packer = keras_nlp.layers.MultiSegmentPacker(\n",
+        "        sequence_length=model.max_sequence_length,\n",
+        "        start_value=tokenizer.token_to_id(\"[CLS]\"),\n",
+        "        end_value=tokenizer.token_to_id(\"[SEP]\"),\n",
+        "    )\n",
+        "    return packer(tokenizer(x))\n",
+        "\n",
+        "token_ids, segment_ids = preprocess([\"The quick brown fox.\"])"
+      ],
+      "metadata": {
+        "id": "iAubWsWj9qtg"
+      },
+      "execution_count": 11,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "metadata": {
+        "id": "-JvyB96k9qtg"
+      },
+      "outputs": [],
+      "source": [
+        "keras_nlp_output = model({\n",
+        "    \"token_ids\": token_ids,\n",
+        "    \"segment_ids\": segment_ids,\n",
+        "    \"padding_mask\": token_ids != 0,\n",
+        "})\n",
+        "\n",
+        "mg_pooled_output, mg_sequence_output = embedding_model(tf.constant([\"The quick brown fox.\"]))"
```

Comment:
nit: These are not mg checkpoints anymore. Maybe `original_pooled_output`? You can search and replace without redoing the notebook.